.circleci/config.yml

```diff
@@ -313,9 +311,8 @@ jobs:
       - run:
           name: Set up Python 3.5
           command: |
-             # 3.5.2 is already installed in circle, so use it
-             pyenv install 3.5.2 || true
-             pyenv global 3.5.2
+             pyenv install 3.5.4 || true
```

Comment:
Circle has 3.5.2 installed by default.  Using 3.5.4 will cause it to download the version of python, which will slow down the build.  Is there a need to test with 3.5.4 instead of 3.5.2?

Reply:
I thought about this, and it's to be consistent with the Docker jobs where we test the "latest" Python 3.5, which is 3.5.4. Given that we've actually seen Girder code break across patch versions of Python (e.g. the hashsum serialization logic), I thought it was significant enough to care about being consistent between our two (Docker vs machine) infrastructures. Once we move to testing Python 3.6, we'll have to install a different version here anyway.